addons/mail/static/src/core/common/chat_hub_model.js

```diff
@@ -121,6 +123,10 @@ export class ChatHub extends Record {
         // folded before opened because if there are too many opened they will be added to folded
         this.folded = toFold.filter((chatWindow) => chatWindow.exists());
         this.opened = toOpen.filter((chatWindow) => chatWindow.exists());
+        this.loaded = true;
+        const cbs = this.loadCbs;
+        this.loadCbs = [];
+        cbs.forEach((fn) => fn());
```

Comment:
This is still executed at each load (which can happen at init and at each localstorage event).
The code for only doing it as init is in static new().

Reply:
The `loadCbs` is cleared each time, so many calls to `load` doesn't recall the old `loadCbs`.

The only flaw I still see is that the code registering the cb doesn't check its cb is already present, so it can be called multiple times on a single load. Do you see any other problem?

Reply:
I don't know if there is a problem, but it's not clean to have this code here when there is a specific place for init code already.